handler.go

```diff
@@ -66,14 +55,16 @@ func (h namedHandler) Handle(ctx context.Context, record slog.Record) error {
 
 	var handler slog.Handler
 	switch config.Format {
-	case FormatText:
-		handler = slog.NewTextHandler(output, opts)
 	case FormatJSON:
 		handler = slog.NewJSONHandler(output, opts)
 	default:
-		// default to slog.TextHandler if no value is set
+		// default to tint.Handler if no value is set
 		// or the set value is invalid
-		handler = slog.NewTextHandler(output, opts)
+		handler = tint.NewHandler(output, &tint.Options{
+			AddSource:   opts.AddSource,
+			Level:       opts.Level,
+			ReplaceAttr: opts.ReplaceAttr,
```

Comment:
question: You're specifying `ReplaceAttr` here but it's no longer defined. Why did you decided to remove it? Do we no longer want the symbol to differentiate between default log KVs and the custom ones?

Reply:
The custom labels were used a crutch to re-order the params in the way we wanted. This handler seems to put them in the exact order we prefer though.

Reply:
It was more than the order. Order was one thing — nice that `tint` gets it right — but making it clear what is extra log KVs what the reason for the `$` prefix. It's not straight forward to test it here so I'll see when you open the PR on defra to update the `corelog` version. I'll trust your jugement :)

Reply:
Actually on second thought we probably want to keep the symbols for some variables.

Reply:
I restored the symbols for `name` and `stack` attributes. 

I also made it so that the tint handler prepends the logger name so the output looks like: `<time> <level> <name> <message> <params...>`

Reply:
I'm curious, why not `time`, `level` and `msg` as well? Maybe there is something I don't see that `tint` is doing in the background but as you see bellow, the `$` symbol helps differentiate between the KVs that are there for every log and the ones that are custom, `PeerId` and `Address`.
```shell
$time="2024-04-22 09:35:44" $name=net $level=INFO $msg="Created LibP2P host" PeerId=12D3KooWE6eTokerm7UXEmLeMxq3P6abgZxWhWsfo6w725yuf78r Address=[/ip4/127.0.0.1/tcp/9171]
```

Reply:
The tint handler doesn't need the `$level`, `$msg`, or `$time` keys since it has this format:

```
Apr 22 09:52:17.833 INF net Created LibP2P host PeerID=12D3KooWE6eTokerm7UXEmLeMxq3P6abgZxWhWsfo6w725yuf78r
```

The JSON handler could keep them though.

```
{"time":"2024-04-22T09:54:04.640625-07:00","level":"INFO","msg":"Created LibP2P host","$name":"net","PeerID":"12D3KooWE6eTokerm7UXEmLeMxq3P6abgZxWhWsfo6w725yuf78r"}
```

Reply:
I've restored the custom keys for the JSON handler.